### PR TITLE
refactor: transactions stats

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Summary

https://app.clickup.com/t/861n8pvx7

requires #496 to be merged first

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my UI changes in light AND dark mode
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: app/Http/Controllers/TransactionsController.php [hunk 1]
```diff
@@ -4,18 +4,42 @@
...
+            'transactionCount' => $data['transaction_count'],
+            'volume'           => BigNumber::new($data['volume'])->toFloat(),
+            'totalFees'        => BigNumber::new($data['total_fees'])->toFloat(),
+            'averageFee'       => BigNumber::new($data['average_fee'])->toFloat(),
         ]);
     }
+
+    private function transactionData(): array
+    {
+        $data = (array) DB::connection('explorer')
```
Comment (ItsANameToo) [inline app/Http/Controllers/TransactionsController.php:29]: shall we cache this for a few minutes? shouldn't matter if the stats lag behind a little

**Review by ItsANameToo — CHANGES_REQUESTED** [review]
- the padding is too large between the statistics and the transaction table, will that be fixed in https://github.com/ArdentHQ/arkscan/pull/495?
- incorrect color for the amounts (design is darker)

Design: 
image

Impl:
image

- design has a line between the stats and the table on 640 and below, the implementation misses that
image

File: app/Http/Controllers/TransactionsController.php (hunk 1 above)
Comment (alexbarnsley) [reply app/Http/Controllers/TransactionsController.php:29]: I'm down for that

**Comment by alexbarnsley** [comment]
Should all be resolved @ItsANameToo. I already had the border between table and stats so I may have done that on the base branch

image

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**